ncclient/devices/clixon.py

```diff
@@ -0,0 +1,31 @@
+"""
+Handler for Clixon based device specific information.
+
+Note that for proper import, the classname has to be:
+
+    "<Devicename>DeviceHandler"
+
+...where <Devicename> is something like "Default", "Nexus", etc.
+
+All device-specific handlers derive from the DefaultDeviceHandler, which implements the
+generic information needed for interaction with a Netconf server.
+
+"""
+
+
+from .default import DefaultDeviceHandler
+
+def iosxr_unknown_host_cb(host, fingerprint):
```

Comment:
Please check why this function is here; it is not used in the device handler below at all.

Also, if it is needed, the name should not include `iosxr`.